src/Http/Http/test/Features/FormFeatureTests.cs

```diff
@@ -625,5 +627,16 @@ private void CompareStreams(Stream streamA, Stream streamB)
                 loops++;
             }
         }
+
+        private static DefaultHttpContext GetDefaultHttpContext()
+        {
+            var httpContext = new DefaultHttpContext
+            {
+                RequestServices = new ServiceCollection()
+                .AddSingleton(Options.Create(new HttpBufferingOptions()))
```

Comment:
We don't register individual options types, AddOptions should be sufficient, it will create the default HttpBufferingOptions.
https://github.com/aspnet/AspNetCore/blob/70b478136a702d2b6061ac9b159029189ad230d0/src/Hosting/Hosting/src/WebHostBuilder.cs#L286